src/gluonnlp/data/__init__.py

```diff
@@ -38,11 +38,12 @@
 from .utils import *
 from .word_embedding_evaluation import *
 from .intent_slot import *
+from .datasetloader import *
 
 
 __all__ = (['batchify'] + utils.__all__ + transforms.__all__ + sampler.__all__
            + dataset.__all__ + corpora.__all__ + sentiment.__all__
            + word_embedding_evaluation.__all__ + stream.__all__ + conll.__all__
            + translation.__all__ + registry.__all__ + question_answering.__all__
            + dataloader.__all__ + candidate_sampler.__all__ + intent_slot.__all__
-           + glue.__all__)  # pytype: disable=attribute-error
+           + glue.__all__ + datasetloader.__all__)  # pytype: disable=attribute-error
```

Comment:
Shall we also update docs/api/modules/data.rst 
I noticed that SplitSampler is also missing there... I forgot to add it